com.avaloq.tools.ddk.checkcfg.core/META-INF/MANIFEST.MF

```diff
@@ -17,6 +17,7 @@ Require-Bundle: org.eclipse.xtext,
  org.eclipse.xtext.xbase;visibility:=reexport,
  org.eclipse.xtext.common.types,
  org.eclipse.jdt.core,
+ io.github.classgraph,

```

Comment:
why do we need this extra dependency in the MANIFEST? There are no other changes than this one in the core plugin